apps/docs/pages/migrating.mdx

```diff
@@ -0,0 +1,63 @@
+# Migrating
+
+## From v1 to v2
+
+Deroll v1 is based on Cartesi Rollups v1, while deroll v2 is based on Cartesi Rollups v2.
+
+Here are the changes you should be aware of when migrating from v1 to v2:
+
+### Input metadata
+
+The input metadata received in an advance handler has changed.
+
+```diff
++chain_id: number;
++app_contract: Address;
+msg_sender: Address;
+-epoch_index: number;
+input_index: number;
+block_number: number;
+-timestamp: number;
++block_timestamp: number;
++prev_randao: Hex;
+```
+
+### Voucher value
+
+Vouchers now have a required hex-encoded bigint `value` field, which is the amount of native token to be sent during voucher execution.
+With this new property withdraw vouchers can now be much simpler.
```

Comment:
Ether withdrawal vouchers, specifically.
For other withdrawal vouchers, like the one shown below, it actually makes them more complicated, unless we make this field optional.